test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,38 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty
+        assertIsNotNull();
+
+        // only one object
+        assertIsNull((Object) null);
+        assertIsNotNull(1);
+        assertIsNotNull("");
+        assertIsNotNull("abc");
+
+        // multiple objects are null
+        assertIsNull(null, null, null);
+
+        // multiple objects with 1 null
+        assertIsNull("1", "a", null, "abx");
+        assertIsNull(1, "a", null, "abx");
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsNull(Object... objects) {
```

Comment:
It's good that you followed the style used by the code nearby.
However, in this case, there is no need to abstract the assertTrue(Utils.isAnyNull(...)) as a separate method. Asserting the result of the method itself is fine.
The existing test uses assertAreUnique() in order to prevent Arrays.asList(..) from being used each time.

Also, this method will return true if any of the arguments is null. So, you might want to name it accordingly.